re_pages/Packages.res

```diff
@@ -450,14 +450,23 @@ let default = (props: props) => {
   | Filtered(pattern) => Resource.applySearch(allResources, pattern)
   }
 
+  let router = Next.Router.useRouter()
+
+  let updateQuery = value =>
+    router->Next.Router.pushObj({
```

Comment:
Would prefer to use the `replace` function on the router that doesn't mess with the browser history.

See my pending [Syntax Widget PR](https://github.com/reason-association/rescript-lang.org/pull/114/files#diff-00b54ebd4efe3d1fb5cbb8ac52f08bd216da3f7baa00f57e7891831679b20040R279) which does a similar thing.